pkg/kubelet/kubelet.go

```diff
@@ -1190,9 +1209,31 @@ func (kl *Kubelet) setupDataDirs() error {
 	if err := os.MkdirAll(kl.getPodsDir(), 0750); err != nil {
 		return fmt.Errorf("error creating pods directory: %v", err)
 	}
+	if err := kl.chownDirForRemappedIDs(kl.getPodsDir()); err != nil {
+		return fmt.Errorf("error chowning pods directory: %v", err)
+	}
 	if err := os.MkdirAll(kl.getPluginsDir(), 0750); err != nil {
 		return fmt.Errorf("error creating plugins directory: %v", err)
 	}
+	if err := kl.chownDirForRemappedIDs(kl.getPluginsDir()); err != nil {
+		return fmt.Errorf("error chowning plugins directory: %v", err)
+	}
+
+	return nil
+}
+
+func (kl *Kubelet) chownDirForRemappedIDs(path string) error {
```

Comment:
since it trips me up each time i look at this, at least add a godoc that its a no-op if user namespace is disabled.